src/components/AddBook.js

```diff
@@ -1,18 +1,38 @@
-const AddBook = () => (
-  <div>
-    <form id="form-add">
-      <input type="text" name="title" placeholder="title" />
-      <input type="text" name="author" placeholder="author" />
+import { useDispatch } from 'react-redux';
+// import your Action Creators
+import { v4 as uuidv4 } from 'uuid';
+import { addBook } from '../redux/books/books';
+
+const AddBook = () => {
+  const dispatch = useDispatch();
+  const submitBookToStore = () => {
+    const parent = document.getElementById('form-add');
+    const newBook = {
+      id: uuidv4(), // make sure it's unique
+      title: parent.title.value,
+      author: parent.author.value,
+      genre: parent.genre.value,
+      progress: 30,
+    };
+    dispatch(addBook(newBook));
+  };
+
+  return (
+    <div>
+      <form id="form-add">
+        <input type="text" name="title" placeholder="title" />
+        <input type="text" name="author" placeholder="author" />
```

Comment:
- [OPTIONAL] Please will be nice to clear your inputs after submitting the form to avoid submitting the same information more than once.